tools/azsdk-cli/Azure.Sdk.Tools.Cli/Commands/SharedCommandGroups.cs

```diff
@@ -1,54 +1,71 @@
+using System.CommandLine;
+
 namespace Azure.Sdk.Tools.Cli.Commands
 {
     public static class SharedCommandGroups
     {
 
         public static readonly CommandGroup AzurePipelines = new(
             Verb: "azp",
-            Description: "Azure Pipelines Tool",
-            Options: []
+            Description: "Azure Pipelines commands",
+            Aliases: ["pipeline"]
         );
 
         public static readonly CommandGroup EngSys = new(
             Verb: "eng",
-            Description: "Internal azsdk engineering system commands",
-            Options: []
-        );
-
-        public static readonly CommandGroup Generators = new(
-            Verb: "generators",
-            Description: "Commands that generate files",
-            Options: []
+            Description: "Internal azsdk engineering system commands"
         );
 
         public static readonly CommandGroup Cleanup = new(
             Verb: "cleanup",
-            Description: "Cleanup commands",
-            Options: []
+            Description: "Cleanup commands"
+        );
+
+        public static readonly CommandGroup Config = new(
+            Verb: "config",
+            Description: "SDK service configuration commands"
         );
 
         public static readonly CommandGroup Log = new(
             Verb: "log",
-            Description: "Log processing commands",
-            Options: []
+            Description: "Log processing commands"
         );
 
         public static readonly CommandGroup Package = new(
-            Verb: "package",
-            Description: "Package management and validation commands",
-            Options: []
+            Verb: "pkg",
+            Description: "Package operations",
+            Aliases: ["package"]
         );
 
-        public static readonly CommandGroup SourceCode = new(
-            Verb: "source-code",
-            Description: "Source code generation and build commands",
-            Options: []
+        public static readonly CommandGroup PackageReadme = new(
+            Verb: "readme",
+            Description: "README operations for SDK packages"
+        );
+
+        public static readonly CommandGroup PackageSample = new(
+            Verb: "sample",
+            Description: "Sample operations for SDK packages"
+        );
+
+        public static readonly CommandGroup PackageTest = new(
+            Verb: "test",
```

Comment:
I know this is the way it is currently, but we have an upper level `azsdk test` and then `azsdk pkg test` what differs, should we combine